adapters/roulax/roulaxtest/exemplary/simple-banner.json

```diff
@@ -0,0 +1,115 @@
+{
```

Comment:
Requesting you to please add the test cases for native, video etc. Also please write the supplemental tests as well.